aw/ViewControllers/Onboard/Onboard1ViewController.swift

```diff
@@ -0,0 +1,34 @@
+import CoreData
+import UIKit
+
+class Onboard1ViewController: UIViewController, MOCViewControllerType {
```

Comment:
These should probably be named rather than numbered.

Reply:
`SplashViewController` and `OnboardLocationViewController` sound good?

Reply:
Yep that's great.

Reply:
Renamed in https://github.com/AmericanWhitewater/aw-ios/pull/3/commits/d4b1c5c8bfe37493882535d375b9b121ef60b1c9